.github/workflows/post_release.yml

```diff
@@ -54,3 +54,34 @@ jobs:
           labels: "release-notes"
           team-reviewers: "smileidentity/mobile"
 
+  generate-docc-reference-docs:
```

Comment:
This change shouldn't show up here. Can you merge main into `feat/offline-mode` so that the diff is correct?

Reply:
I see I have some rebase issues